src/app/theme/theme.component.ts

```diff
@@ -58,6 +61,21 @@ export class ThemeComponent implements OnInit {
     this.themeService.titleColor = '#1a0dab';
     this.themeService.linkColor = '#006621';
     this.themeService.descriptionColor = '#545454';
+    (document.getElementsByClassName("infobox")[0] as HTMLElement).style.color = '#252f41';
```

Comment:
You can put all the code that alter the other elements in different functions and call those functions in `nightTheme()` and `defaultTheme()`.

Reply:
Such a pretty good idea. I'll implement it right now. :)